modules/replicator/build.gradle

```diff
@@ -29,6 +29,9 @@ dependencies {
     implementation project(':ignite-core')
     implementation project(':ignite-raft')
     implementation project(':ignite-raft-api')
+    implementation project(':ignite-metastorage-api')
```

Comment:
That seems incorrect. Replicator module is a generic abstraction that might be implemented by partition replicas or  meta storage ones, thus it should not depend on neither metastorage-api nor distribution-zones nor storage-api. Not sure about latter one though.